.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 build/
 .vscode/
 *.jshintrc
+.env
```

Comment:
Please commit .env and .staging.env file along with instruction on how to build/launch app on particular environment.